crates/valence_anvil/src/lib.rs

```diff
@@ -17,133 +17,75 @@
     clippy::dbg_macro
 )]
 
-use std::collections::hash_map::Entry;
-use std::collections::{BTreeMap, HashMap, HashSet};
 use std::fs::File;
 use std::io::{ErrorKind, Read, Seek, SeekFrom};
 use std::num::NonZeroUsize;
 use std::path::PathBuf;
-use std::thread;
 
-use bevy_app::prelude::*;
-use bevy_ecs::prelude::*;
+#[cfg(feature = "bevy")]
+pub use bevy::*;
 use byteorder::{BigEndian, ReadBytesExt};
 use flate2::bufread::{GzDecoder, ZlibDecoder};
-use flume::{Receiver, Sender};
 use lru::LruCache;
+use thiserror::Error;
 use tracing::warn;
```

Comment:
:thinking: is this used anywhere? I'm surprised there's no warning. I guess the builtin `warn` attribute is getting the compiler confused.

Reply:
Line 44

Reply:
Sorry, I'm pointing at `use tracing::warn;` here.